src/main/resources/workbench.properties

```diff
@@ -39,4 +39,7 @@ atomikos.transaction.timeout=${atomikos.transaction.timeout}
 
 # path to the GraphViz' dot executable
 graphviz.executable.path=${graphviz.executable.path}
-pagedresult.max.page.size=${pagedresult.max.page.size}
\ No newline at end of file
+pagedresult.max.page.size=${pagedresult.max.page.size}
+
+# Germplasm Search
+germplasm.search.old.available=false
```

Comment:
rename to:
old.germplasm.search.enabled
can we add something more descriptive? 
# Enables Old Germplasm Search Feature. This property will be removed in v18 and old module will not be available anymore
@cuenyad